infra/gcp/namespaces/namespace-user-role-binding.yml

```diff
@@ -0,0 +1,14 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: namespace-user
+  namespace: {{namespace}}
+subjects:
+  - kind: Group
+    name: k8s-infra-rbac-{{namespace}}@kubernetes.io
+    namespace: {{namespace}}
```

Comment:
Docs say:

```
If the object kind is non-namespace, such as "User" or "Group", and this value is not empty the Authorizer should report an error. 
```

So I think we don't want this?

Reply:
Agree :-)

Reply:
Fixed